Docs/user_guide/coordinate_systems.md

```diff
@@ -0,0 +1,269 @@
+axes:
+
+![coordinate_systems](../_static/images/coordinate_systems.png)
+
+Each coordinate system serves one purpose and represents their data in
+different ways.
+
+Anatomy image based on an [image shared by the My MS organization](https://my-ms.org/mri_planes.htm).
+
+Note that Chand John of Stanford created a [detailed presentation about the way coordinates are handled in Slicer](https://www.na-mic.org/w/img_auth.php/3/3f/Coordinate_Systems_Demystified.ppt).
+
+### World coordinate system
+
+The world coordinate system is typically a Cartesian coordinate system in
+which a model (e.g. a MRI scanner or a patient) is positioned. Every model has
+its own coordinate system but there is only one world coordinate system to
+define the position and orientation of each model.
+
+### Anatomical coordinate system
+
+The most important model coordinate system for medical imaging techniques is
+the anatomical space (also called patient coordinate system). This space
+consists of three planes to describe the standard anatomical position of a
+human:
+
+- the *axial plane* is parallel to the ground and separates the head
+  (Superior) from the feet (Inferior).
+- the *coronal plane* is perpendicular to the ground and separates the front
+  from (Anterior) the back (Posterior).
+- the *sagittal plane* separates the Left from the Right.
+
+From these planes it follows that all axes have their notation in a positive
+direction (e.g. the negative Superior axis is represented by the Inferior axis).
+
+The anatomical coordinate system is a continuous three-dimensional space in
+which an image has been sampled. In neuroimaging, it is common to define this
+space with respect to the human whose brain is being scanned. Hence, the 3D
+basis is defined along the anatomical axes of anterior-posterior,
+inferior-superior, and left-right.
+
+However different medical applications use different definitions of this 3D
+basis. Most common are the following bases:
+
+- LPS (Left, Posterior, Superior) is used in DICOM images
+
+$$
+LPS = \begin{Bmatrix}
+\text{from right towards left} \\
+\text{from anterior towards posterior} \\
+\text{from inferior towards superior}
+\end{Bmatrix}
+$$
+
+- RAS (Right, Anterior, Superior) is similar to LPS with the first two axes
+  flipped
+
+$$
+RAS = \begin{Bmatrix}
+\text{from left towards right} \\
+\text{from posterior towards anterior} \\
+\text{from inferior towards superior}
+\end{Bmatrix}
+$$
+
+Thus, the only difference between the two conventions is that the sign of the
+first two coordinates is inverted.
+
+Both bases are equally useful and logical. It is just necessary to know to
+which basis an image is referenced.
+
+### Image coordinate system
+
+The image coordinate system describes how an image was acquired with respect
+to the anatomy. Medical scanners create regular, rectangular arrays of points
+and cells which start at the upper left corner. The $$i$$ axis increases to
```

Comment:
```suggestion
and cells which start at the upper left corner. The $i$ axis increases to
```

Assuming this is an lined "equation", it should be enclosed with a single `$`

Reply:
Done in [17cea93](https://github.com/Slicer/Slicer/commit/17cea93d3bc6b3279333b3f3a71728394a55bd7c).